docs/reference/mapping/types/text.asciidoc

```diff
@@ -128,3 +135,31 @@ The following parameters are accepted by `text` fields:
     Whether term vectors should be stored for an <<mapping-index,`analyzed`>>
     field. Defaults to `no`.
 
+[[index-prefix-config]]
+==== Index Prefix configuration
+
+To configure the index prefix field, use the following syntax.  Either or both
+of `min_chars` and `max_chars` may be excluded.
+
```

Comment:
I think we should re-explain what it does here and also make it clear that both min_chars and max_chars are inclusive?